indico/modules/events/editing/client/js/components/FileManager/reducer.js

```diff
@@ -119,6 +131,7 @@ export default combineReducers({
       case actions.MARK_DELETED:
       case actions.UNDELETE:
       case actions.REVERT:
+      case actions.UPLOAD_ERROR:
```

Comment:
probably not needed since we use `isDirty` to trigger the `useEffect` that calls `onChange`, and an error does not change the value